ethtx/providers/semantic_providers/const.py

```diff
@@ -0,0 +1,18 @@
+#  Copyright 2021 DAI Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from enum import Enum
+
+
+class MongoCollections(str, Enum):
```

Comment:
Why do it inherit after enum and str?

Reply:
Enum for define a set of named constants. And mixins, coz of that: https://docs.python.org/3/library/enum.html#others